### PR TITLE
Docker 2.13

### PR DESCRIPTION
**Pull request by davidneudorfer**
cc @smit1678 and @bgirardot 

If you download [Docker for Mac](https://docs.docker.com/docker-for-mac/) you can follow the steps in DOCKER.md to deploy TM2 locally. Before the restore step if you browse to http://localhost:6543/ before the database restore step you'll see a working copy of TM2.

Restoring the DB returns 1900 errors from `pg_restore` and the following from the app server.
`ProgrammingError: (ProgrammingError) permission denied for relation task`

### CODE REVIEW

**Comment by bgirardot** [comment]
@pgiraud Do you have any idea what might be going on here?

File: setup.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
     'gitversion',
     'APScheduler==3.0.3',
     'python-oembed == 0.2.1',
+    'html5lib<0.99999999'
```
Comment (pgiraud) [inline setup.py]: Can you please elaborate why you need to install html5lib?

**Comment by pgiraud** [comment]
> Restoring the DB returns 1900 errors from pg_restore and the following from the app server.
> ProgrammingError: (ProgrammingError) permission denied for relation task

This has to be confirmed but as far as I remember, the user that Drazen chose to use for the database owner is not "www-data".

In order to restore a database from the production server locally, here's the command is usually use:
```shell
pg_restore -d osmtm the_osmtm2_xxx.dmp -Ox --role=www-data
```

File: setup.py (hunk 1 above)
Comment (bgirardot) [inline setup.py]: It is required by bleach, and a very specific version so we have to install it specifically.

env/lib/python2.7/site-packages/bleach-1.4.3.dist-info/METADATA:Requires-Dist: html5lib (>=0.999,<0.99999999)

File: Dockerfile.app [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ADD setup.cfg /opt/app/setup.cfg
+ADD README.md /opt/app/README.md
+ADD CHANGELOG.md /opt/app/CHANGELOG.md
+
+WORKDIR /opt/app
+
+RUN easy_install virtualenv
+RUN virtualenv --no-site-packages env
+RUN ./env/bin/pip install -r requirements.txt
+RUN ./env/bin/pip install html5lib==0.999
```
Comment (pgiraud) [inline Dockerfile.app]: This should already be taken into account by the line added in `setup.py`.

File: setup.py (hunk 1 above)
Comment (pgiraud) [inline setup.py]: Did you get any error?

The version of `html5lib` installed without this line added is `0.9999999` (7 x 9). It matches the requirements from bleach (8 x 9).
0.9999999 < 0.99999999 == true

So I still don't understand why we need to install html5lib specifically.

**Comment by davidneudorfer** [comment]
@pgiraud Thank you. `pg_restore -d osmtm /srv/osmtm2.dmp -Ox --role=www-data` was what I was missing.

**Comment by bgirardot** [comment]
Hi @davidneudorfer , does this support both a development deploy and a production deploy?

**Pull request merged**